data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Agumon', 'feb-3-2020', 0, true, 10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (2, 'Gabumon', 'Nov-15-2018', 2, true, 8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (3, 'Pikachu', 'Jan-7-2021', 1, false, 15.04);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (3, 'Devimon', 'May-12-2017', 5, true, 11);
+
```

Comment:
- [ ] There's a spelling error in your data.sql file. You declared `escape_attempt` in your schema file NOT `escape_attempts`. Kindly remove the `s` to fix the error.

### Error
![error](https://github.com/clin2on3mun/VetClinic-database/assets/26597432/f2f10ee1-2979-4bd8-a70b-90be610fff5a)